### PR TITLE
New control scheme

### PR DESCRIPTION
**Pull request by SebZanardo**
In progress, remapping controls for teleop
- [x] Bind shooter
- [x] Test reset heading at podium is correct

### CODE REVIEW

File: components/led.py [hunk 1]
```diff
@@ -72,10 +72,18 @@ def want_note(self) -> None:
         self.set_colour(HsvColour.MAGENTA.value)
         self.set_state(PatternState.FLASH)
 
-    def holding_note(self) -> None:
+    # def holding_note(self) -> None:
```
Comment (fotnitfpro000) [inline components/led.py:75]: will the holding_note led state be used/necessary?

Comment (SebZanardo) [reply components/led.py:75]: No, I don't think so. It'll be switching to in_range and out_of_range states once holding the note

File: components/intake.py [hunk 2]
```diff
@@ -9,7 +9,7 @@
 
 
 class IntakeComponent:
-    motor_speed = tunable(0.4)
+    motor_speed = tunable(0.6)
```
Comment (mlists) [inline components/intake.py:12]: This PR should not change the intake

File: components/chassis.py [hunk 3]
```diff
@@ -221,8 +221,12 @@ class ChassisComponent:
     do_smooth = magicbot.tunable(True)
     swerve_lock = magicbot.tunable(False)
 
+    # TODO: Read from positions.py once autonomous is finished
     RED_TEST_POSE = Pose2d(15.1, 5.5, math.pi)
     BLUE_TEST_POSE = field_flip_pose2d(RED_TEST_POSE)
+    stage_width = 16.541748046875
+    RED_PODIUM = Pose2d(Translation2d(3.367, 4.08455), Rotation2d(math.pi))
+    BLUE_PODIUM = Pose2d(Translation2d(stage_width - 3.367, 4.08455), Rotation2d(0))
```
Comment (mlists) [inline components/chassis.py:229]: We already have a utility function to do this flipping

Comment (mlists) [inline components/chassis.py:229]: I think we should move all these poses into positions.py

**Review by outsidermm — APPROVED** [review]
LGTM

File: robot.py [hunk 4]
```diff
@@ -111,20 +106,33 @@ def teleopPeriodic(self) -> None:
...
-        if self.gamepad.getXButton():
+        if self.gamepad.getBButton():
             self.chassis.reset_yaw()
 
-        # stop rumble after time
-        if self.rumble_timer.hasElapsed(self.rumble_duration):
-            self.gamepad.setRumble(wpilib.XboxController.RumbleType.kBothRumble, 0)
+        # Reset Odometry
+        if self.gamepad.getStartButtonPressed():
+            self.chassis.reset_odometry()
```
Comment (auscompgeek) [inline robot.py:114]: One of these simply reads the button state, whereas the other reads the rising edge of the button. Should these be consistent?

File: controllers/climber.py [hunk 5]
```diff
@@ -5,6 +5,7 @@
 class Climber(StateMachine):
     climber_component: ClimberComponent
     should_climb = will_reset_to(False)
+    extended = False
```
Comment (auscompgeek) [inline controllers/climber.py:8]: nit: initialise this as an instance attribute

**Review by auscompgeek — APPROVED** [review]
LGTM

**Pull request merged**